src/coverlet.core/Helpers/FilePathHelper.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Coverlet.Core.Abstractions;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+
+
+namespace Coverlet.Core.Helpers
+{
+    internal class FilePathHelper : IFilePathHelper
```

Comment:
I think that extract a service is too much for a path helper...I mean we won't inject different implementation there are no different way to get that path from file system.
Should be enough add a new base class `abstract ReporterBase` and add this two method as `protected` and inherit the reporter that needs this logic.

Reply:
So only the reporter classes that need this behavior will inherit from ReporterBase and rest of them will continue to implement IReporter?
Why not inherit every reporter class from ReporterBase to keep things consistent?

Reply:
Ok I won't oppose, the important things is that we have less cose as possible to maintain, let's go for base class inherited by reporters

Reply:
Cool. I am going to close this pr and open a new one with base class design.